napalm_logs/config/fortinet/FORWARD_TRAFFIC.yml

```diff
@@ -0,0 +1,43 @@
+# Logs files identified as type=traffic and subtype=forward (FortiOs 5.4).
+#
+# <189>date=2019-04-09 time=04:27:29 devname=fw01 devid=FG800D0123456789 logid=0000000013 type=traffic subtype=forward 
+# level=notice vd=root srcip=1.1.1.1 srcport=19982 srcintf="port1" dstip=10.10.10.10 dstport=179 dstintf="port3" 
+# poluuid=d4954a18-3b72-51e9-1163-77fc436ef3c9 sessionid=1768791677 proto=6 action=timeout policyid=34 policytype=policy 
+# dstcountry="Reserved" srccountry="Reserved" trandisp=noop service="BGP" duration=25 sentbyte=300 rcvdbyte=0 sentpkt=5 
+# rcvdpkt=0 appcat="unscanned" crscore=5 craction=262144 crlevel=low
+#
+messages:
+  - error: 'FORWARD_TRAFFIC'
+    tag: "subtype=forward"
+    values:
+      level: ([^ ]+)
+      vd: ([^ ]+)
+      srcip: ([^ ]+)
+      srcport: ([^ ]+)
+      srcintf: ([^ ]+)
+      dstip: ([^ ]+)
+      dstport: ([^ ]+)
+      dstintf: ([^ ]+)
+      poluuId: ([^ ]+)
+      sessiondId: ([^ ]+)
+      protocolId: ([^ ]+)
+      action: ([^ ]+)
+      miscData: (.*)
+    line: '{level} {vd} {srcip} {srcport} {srcintf} {dstip} {dstport} {dstintf} {poluuId} {sessiondId} {protocolId} {action} {miscData}'
```

Comment:
The variables in this line should only be the data after each `=`, i.e `'level={level}...`

Reply:
Ok i'm going make the change in this way